mattermost/mattermost
# Feature/create legalhold test

#### Summary

**This is to test that an admin user can create a legal hold**

Verify that an admin user can create a new legal hold 

Test Scenario

- On the legal hold plugin page

- Click Create legal hold button

- Enter name

- Enter a start date

- Select user

- Check that admin is able to create a legal hold successfully

Proof of work

#### Screenshots
<img width="985" alt="Screenshot 2024-10-29 at 20 50 37" src="https://github.com/user-attachments/assets/bcf90a66-58a7-4227-b61d-26b508c82138">

```release-note

```
